connection.cpp

```diff
@@ -159,12 +202,12 @@ void Connection::leaveRoom(Room* room)
     job->start();
 }
 
-void Connection::getMembers(Room* room)
-{
-    RoomMembersJob* job = new RoomMembersJob(d->data, room);
-    connect( job, &RoomMembersJob::result, d, &ConnectionPrivate::gotRoomMembers );
-    job->start();
-}
+//void Connection::getMembers(Room* room)
+//{
+//    RoomMembersJob* job = new RoomMembersJob(d->data, room);
+//    connect( job, &RoomMembersJob::result, d, &ConnectionPrivate::gotRoomMembers );
+//    job->start();
+//}
```

Comment:
On the other hand - probably, this very piece of code is really unneeded - it already gets rotten, similar to the rest.